dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -1,27 +1,27 @@
 # For testing and analyzing code.
-mypy==0.701.*
-pylint==2.3.*
-pytest==3.8.*
-pytest-cov==2.5.*
-pytest-benchmark==3.2.*
-yapf==0.27.*
+mypy~=0.701
+pylint~=2.3
+pytest~=3.8
+pytest-cov~=2.5
+pytest-benchmark~=3.2
+yapf~=0.27
```

Comment:
I think you need to do `yapf~=0.27.0` instead (https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean).  Same for most of the others.

Reply:
done